plugins/node/opentelemetry-instrumentation-pg/src/internal-types.ts

```diff
@@ -47,10 +47,11 @@ export type PgPoolCallback = (
 ) => void;
 
 export interface PgPoolOptionsParams {
-  database: string;
-  host: string;
-  port: number;
-  user: string;
+  database?: string;
+  host?: string;
+  port?: number;
+  user?: string;
```

Comment:
Actually, there four properties are not mandatory, so I changed them optional.

See:
- https://github.com/brianc/node-postgres/blob/pg-pool%403.5.2/packages/pg-pool/index.js#L64
- https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/pg-pool/index.d.ts#L12
- https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/pg/index.d.ts#L17-L21